ui_framework/src/components/icon/_icon.scss

```diff
@@ -0,0 +1,27 @@
+@include component('kuiIcon') {
+  display: inline-block;
+
+  @include modifier('xSmall') {
```

Comment:
Can delete anything below 16px. We never go below 16px and I'd rather not even allow it. They become unreadable.